en/contributors/core-team.md

```diff
@@ -141,6 +141,8 @@ The core team is composed of members who make frequent contributions to the proj
       <td>Poland</td>
       <td>
           <ul>
+              <li>C#>
```

Comment:
Looks like you have a malformed tag here

Reply:
Upsy 😬 Done!

Reply:
Almost there!!! One more character to remove 🙂.

Like C#**>** is more than anything but we don't need to mentioned it there.

<img width="230" alt="image" src="https://github.com/user-attachments/assets/cd0dba1f-299e-4d0a-95f6-bc3776e276d2">

Reply:
Done!